ocs_ci/framework/main.py

```diff
@@ -94,10 +95,16 @@ def main(argv=None):
     init_ocsci_conf(arguments)
     pytest_logs_dir = utils.ocsci_log_path()
     utils.create_directory_path(framework.config.RUN['log_dir'])
+    launch_name = f"Tier run from {getuser()} RunId: {framework.config.RUN['run_id']}"
     arguments.extend([
         '-p', 'ocs_ci.framework.pytest_customization.ocscilib',
         '-p', 'ocs_ci.framework.pytest_customization.marks',
         '-p', 'ocs_ci.framework.pytest_customization.reports',
         '--logger-logsdir', pytest_logs_dir,
+        '--rp-launch', launch_name,
     ])
+    if '--reportportal' not in arguments:
```

Comment:
Doesn't this make running without posting to report portal impossible? I'm not sure I like the idea of forcing users to post to report portal if they don't want to use this functionality.

Reply:
I would like to see this enabled for all runs, we currently dont have visibility outside jenkins. For accurate failure rate we should see results from across. The setting ignores failures due to reportportal issues but if there are issues its not hard to disable.

Reply:
This forces every execution to push to report portal, even local executions. Would it be acceptable if we had `--report-portal` configured by default for all of our jenkins jobs instead?

Reply:
I will say, lets load our DB, we can revisit this in a weeks time , there is nothing wrong in having dashboard integrated by default.

Reply:
And would you be able to filter out the local runs in the report portal? I can imagine that my local runs which uses not yet comited code for test development purposes could mess with the data we see there.

Reply:
yeah we can have multiple filters, so its easy to filter things out, Personally played with tons of data in ceph. Right now this is disabled by default so it requires explicit option for now.